src/components/pages/doc.yml

```diff
@@ -10,3 +10,6 @@ documentMetadataUpdated:
 commentsPanelTitle:
   en: You are viewing the document page in "comments" mode
   de: Du befindest dich im Kommentar-Modus
+commentsHeader:
+  en: Comment topics
+  de: Kommentare-Themen
```

Comment:
No double-plural here, just "Kommentar-Themen"

Reply:
Corrected, thank you